content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
```

Comment:
```suggestion
Follow this guide to deploy a [multicluster service mesh](/docs/ops/deployment/deployment-models/#multiple-clusters) spanning one more more clusters and then connect a virtual machine to it.
```

Reply:
thanks @frankbu for the suggestion. I have applied it unmodified in my local tree and will submit a new commit.